libvast/src/data.cpp

```diff
@@ -448,6 +448,15 @@ caf::error convert(const data& d, caf::config_value& cv) {
   return caf::visit(f, d);
 }
 
+const data* get_if(const record* xs, std::string_view name) {
+  // Access the key directly unless the user specified a dot-separated path.
+  auto i = xs->find(name);
+  if (i == xs->end())
+    return nullptr;
+  // We can't simply return the result here, because it might be a pointer.
+  return &i->second;
```

Comment:
```suggestion
return i == xs.end() ? nullptr : &i->second;
```

We have this trichotomy pretty frequently. Let's make it a pattern.